lib/astc_codec.cpp

```diff
@@ -291,38 +293,121 @@ astcVkFormat(ktx_uint32_t block_size, bool sRGB) {
 
 /**
  * @memberof ktxTexture
- * @ingroup writer
+ * @internal
+ * @ingroup reader
  * @~English
- * @brief Creates valid ASTC encoder action from string.
+ * @brief       Should be used to get uncompressed version of ASTC VkFormat
  *
- * @return      Valid astc_profile from string
+ * The decompressed format is calculated from coressponding ASTC format. There are
```

Comment:
Another s/coressponding/corresponding/. I didn't spot it before.

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Software/pull/967/commits/42c08d41f30910ced10ffc9d2170cfb547d2bfb8